content/topics/intro-to-ajax/_index.md

```diff
@@ -3,8 +3,18 @@ _db_id: 146
 content_type: topic
 ready: true
 tags:
-- ajax
+  - ajax
 title: Introduction to Ajax
 ---
 
-[Ajax Tutorial](https://www.codementor.io/sheena/ajax-tutorial-web-development-du107rzaq?referral=sheena-kvo1e6ewh)
\ No newline at end of file
+Ajax, short for Asynchronous JavaScript and XML, revolutionized web development by enabling seamless asynchronous communication between a web application's frontend and server. This technology allows specific parts of a webpage to be updated dynamically without requiring a complete refresh, resulting in faster and more responsive user experiences. By utilizing JavaScript to send and receive data in the background, Ajax has significantly contributed to the interactivity and real-time capabilities of modern web applications.
+
+Have a look at the following tutorial to see how ajax functionality can be achieved with JQuery:
+
+[Ajax Tutorial with JQuery](https://www.codementor.io/sheena/ajax-tutorial-web-development-du107rzaq?referral=sheena-kvo1e6ewh)
+
+Ajax functionality can also be achieved with vanilla JavaScript. This involves using the XMLHttpRequest object to send asynchronous HTTP requests to the server and handle responses without requiring additional libraries. Through this approach, developers can update webpage content dynamically and enhance user interactions without relying on external frameworks.
+
+Have a look at the following tutorial to see how ajax functionality can be achieved with vanilla JavaScript:
+
+[Ajax Tutorial with vanilla JS](https://developer.mozilla.org/en-US/docs/Web/Guide/AJAX)
```

Comment:
Another way of making "ajax" calls is with `fetch` or a library like axios, maybe that can be added under the "other ways" section. The section can be named whatever you feel is a good match

Reply:
If one reads this Mozilla write-up, they will come across information on the fetch web API on the "Related technologies headings"